e2e-tests/cypress/integration/move-skills/move_skills_modal_spec.js

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright 2020 SkillTree
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+describe('Move Skills Modal Tests', () => {
+
+    beforeEach(() => {
+        cy.createProject(1);
+        cy.createSubject(1, 1);
+        cy.createSkill(1, 1, 1);
+    });
+
+    it('no destination', () => {
+        cy.visit('/administrator/projects/proj1/subjects/subj1');
+        cy.get('[data-cy="skillSelect-skill1"]')
+            .click({ force: true });
+        cy.get('[data-cy="skillActionsBtn"]')
+            .click();
+        cy.get('[data-cy="skillMoveBtn"]')
+            .click();
+
+        cy.contains('There are no Subjects or Groups that this skill can be moved to.');
+
+        cy.get('[data-cy="destinationList"]')
+            .should('not.exist');
+        cy.get('[data-cy="reuseButton"]')
+            .should('be.disabled');
+        cy.get('[data-cy="closeButton"]')
+            .should('be.enabled');
+        cy.get('[data-cy="okButton"]')
+            .should('not.exist');
+        cy.get('[data-cy="destListPagingControl"]')
+            .should('not.exist');
+        cy.get('[data-cy="reuseSkillsModalStep1"]')
+            .contains('No Destinations');
+    });
+
+    it('multiple skills - some are already reused', () => {
+        cy.createSubject(1, 2);
+        cy.createSkill(1, 1, 3);
+        cy.createSkill(1, 1, 4);
+        cy.createSkill(1, 1, 5);
+        cy.reuseSkillIntoAnotherSubject(1, 1, 2);
+        cy.reuseSkillIntoAnotherSubject(1, 5, 2);
+        cy.visit('/administrator/projects/proj1/subjects/subj1');
+        cy.get('[data-cy="selectAllSkillsBtn"]')
+            .click();
+        cy.get('[data-cy="skillActionsBtn"]')
+            .click();
+        cy.get('[data-cy="skillMoveBtn"]')
+            .click();
+        cy.get('[data-cy="reuseButton"]')
+            .should('be.disabled');
+        cy.get('[data-cy="closeButton"]')
+            .should('be.enabled');
+        cy.get('[data-cy="okButton"]')
+            .should('not.exist');
+
+        cy.get('[ data-cy="reuseSkillsModalStep1"] [data-cy="selectDest_subjsubj2"]')
+            .click();
+        cy.get('[ data-cy="reuseSkillsModalStep2"]')
+            .contains('2 skills will be moved to the [Subject 2] subject.');
+        cy.get('[ data-cy="reuseSkillsModalStep2"]')
```

Comment:
If the user selects skills that are "reused" to move, is the message they are shown "selected skills have already been reused"? If so that doesn't seem like it would be very clear.

Reply:
:+1: